analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Branching.LearnConstraint.cs

```diff
@@ -386,11 +386,10 @@ public void Branching_LearnsObjectConstraint_ConstantPattern_False(string expres
         }
 
         [DataTestMethod]
-        [DataRow("arg is 42")]
-        [DataRow("arg is System.ConsoleKey.Enter")]    // Enum
-        public void Branching_LearnsObjectConstraint_ConstantPattern_ValueTypes(string expression)
+        [DataRow("arg is 42", "int")]
```

Comment:
```suggestion
        [DataRow("arg is 42", "int")]
        [DataRow("arg is 42", "TStruct")]
```

Reply:
I've added them, but we're going out of scope of declaration pattern in this PR